src/views/Learn.vue

```diff
@@ -1,12 +1,12 @@
 <template>
   <div >
     <FadeTeaser>
-      <div class="px-0 py-40 flex text-center justify-center">
```

Comment:
This should also not be done here, since TeaserHeader will be added in my PR, centralizing such behaviour.